langchain/agents/conversational_chat/base.py

```diff
@@ -131,11 +132,13 @@ def from_llm_and_tools(
         human_message: str = SUFFIX,
         input_variables: Optional[List[str]] = None,
         output_parser: Optional[BaseOutputParser] = None,
+        template_tool_response: Optional[str] = TEMPLATE_TOOL_RESPONSE,
         **kwargs: Any,
     ) -> Agent:
         """Construct an agent from an LLM and tools."""
         cls._validate_tools(tools)
         _output_parser = output_parser or AgentOutputParser()
+        cls.template_tool_response = template_tool_response
```

Comment:
i would pass this into the class constructor call (one line down) rather than set here